packages/rest/src/parser.ts

```diff
@@ -122,3 +125,50 @@ function buildOperationArguments(
   if (requestBodyIndex > -1) paramArgs.splice(requestBodyIndex, 0, body);
   return paramArgs;
 }
+
+export function paramCoerce(data: string, schema?: SchemaObject | ReferenceObject): any {
```

Comment:
Let's move this function to a new file please. I am expecting it to grow in size as we will deal with edge cases.

Reply:
Rename it to `coerceParam`?

Reply:
+1 for coerceParam, I would even call it `coerceParameter`. It's our convention for function/method names to start with a verb.